scripts/components/material/guild.js

```diff
@@ -71,7 +71,11 @@ JoinServer.propTypes = {
 
 function ShowMore(props) {
   const classes = useStyles()
-  const { expanded } = props
+  const { expanded, emojiCount } = props
+
+  if (emojiCount !== undefined && emojiCount <= RANDOM_SAMPLE_SIZE) {
```

Comment:
I would be extremely interested to know where the emojiCount is undefined seeing as you read it from the length

Reply:
I don't remember why I have added that check, I'll remove it